pkg/frontend/kubeactions/kubeactions.go

```diff
@@ -160,6 +163,63 @@ func (ka *kubeactions) List(ctx context.Context, oc *api.OpenShiftCluster, kind,
 	return ul.MarshalJSON()
 }
 
+func (ka *kubeactions) applyOne(ctx context.Context, dyn dynamic.Interface, grs []*restmapper.APIGroupResources, un *unstructured.Unstructured) error {
+	if strings.EqualFold(un.GetKind(), "secret") { // TODO do we need this?
+		return fmt.Errorf("access to secrets is forbidden")
+	}
+	gvrs := ka.findGVR(grs, un.GetKind())
+	if len(gvrs) == 0 {
+		return fmt.Errorf("Kind %s not found", un.GetKind())
+	}
+	if len(gvrs) > 1 {
+		return fmt.Errorf("Kind %s did not uniquely match one GRV", un.GetKind())
+	}
+	gvr := gvrs[0]
+
+	ka.log.Debugf("appling kube object %s %s %s ", un.GetKind(), un.GetNamespace(), un.GetName())
+
+	_, err := dyn.Resource(*gvr).Namespace(un.GetNamespace()).Create(un, metav1.CreateOptions{})
+	if err == nil || !errors.IsAlreadyExists(err) {
+		return err
+	}
+
+	return retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		_obj, err := dyn.Resource(*gvr).Namespace(un.GetNamespace()).Get(un.GetName(), metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		un.SetResourceVersion(_obj.GetResourceVersion())
+		_, err = dyn.Resource(*gvr).Namespace(un.GetNamespace()).Update(un, metav1.UpdateOptions{})
+		return err
+	})
+}
+
+func (ka *kubeactions) Apply(ctx context.Context, oc *api.OpenShiftCluster, body []byte) error {
+	dyn, grs, err := ka.getClient(oc)
+	if err != nil {
+		return err
+	}
+
+	obj := &unstructured.Unstructured{}
+	err = obj.UnmarshalJSON(body)
+	if err == nil {
+		return ka.applyOne(ctx, dyn, grs, obj)
+	}
+	objs := &unstructured.UnstructuredList{}
+	err = objs.UnmarshalJSON(body)
+	if err != nil {
+		return err
+	}
+	for objIx := range objs.Items {
+		err = ka.applyOne(ctx, dyn, grs, &objs.Items[objIx])
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
let's not support this for now, I think it's too powerful and the error handling is too complex.